test-refactoring/avx2.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/sh
```

Comment:
@JohanMabille We're not running these tests during CI, but I found them super useful during the portage so I'm checking them in